### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by ghost** [birros/web-archives#12]
sorry for asking but do you still work on this project?

**Comment by birros** [comment]
Good question, the project is in maintenance, I don't add any features, I just update the dependencies every time a new version of the GNOME runtime is released.

Another project (private for the moment) is taking up all my time, related to problems I encountered while designing web archives regarding the synchronization of archives, favorites and history between my devices.

I also doubt I will be able to continue the development of this project, in this form. If I have to rewrite it, I'll use another language, Vala is interesting but the ecosystem is too limited, moreover GNOME abandons Vala in favor of Rust.

**Comment by kelson42** [comment]
@birros Join us to improve new Kiwix Desktop, you are more than welcome and help would be really welcome :)

**Comment by ghost** [comment]
Good to know but sad to see it being abandoned and since this is one of project that really like because it has night mode,  you can add your own zim file, it works better on manjaro my distro.

I do like kiwix as well but it missing those features.

**Comment by kelson42** [comment]
@tlmnot It is not really the right place to talk about that... but have you open tickets? For example, you can perfectly had your own ZIM files!

**Comment by ghost** [comment]
Fair point you got there but I will post support ticket later today, but the message was more met to @birros

**Comment by tlmnot** [comment]
Fair point you got there but I will post support ticket later today, but the message was more met to @birros

**Comment by ghost** [comment]
Speaking of @birros i have another question, i can provide dutch translation but do you need it?

**Comment by birros** [comment]
@tlmnot All contributions are welcome 👍

However, open a new ticket about the Dutch translation, or better, create a pull request. You can take your inspiration from that pull request: [Provide Spanish translation #4](https://github.com/birros/web-archives/pull/4)

You can use [Poedit](https://github.com/vslavik/poedit) to edit the translation file.

**Comment by ghost** [comment]
Great to hear, i will start making the dutch translation.

**Comment by tlmnot** [comment]
Great to hear, i will start making the dutch translation.

**Comment by TheEvilSkeleton** [comment]
> Good question, the project is in maintenance, I don't add any features, I just update the dependencies every time a new version of the GNOME runtime is released.

> All contributions are welcome 👍

@birros hey, seeing that the Flatpak hasn't been updated in two years, are you still open for contributions that are related to updating these dependencies?

**Comment by birros** [comment]
> [@birros](https://github.com/birros) hey, seeing that the Flatpak hasn't been updated in two years, are you still open for contributions that are related to updating these dependencies?

Of course! I was planning on updating it a little after the release of GNOME 48, but if you want to do it, you're welcome 🙂
